### PR TITLE
Feat e2e test cortexso hub

### PR DESCRIPTION
**Pull request by hiento09**
## Describe Your Changes

- Add e2e test cortexso model hub

## Fixes Issues

- Related ticket janhq/models#56 
- Closes #

## Self Checklist

- [ ] Added relevant comments, esp in complex areas
- [ ] Updated docs (for bug fixes / features)
- [ ] Created issues for follow-up changes or refactoring needed

### CODE REVIEW

File: .github/workflows/test-cortexso-model-hub.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
+name: Test cortexso Model Hub
+
+on:
+  schedule:
+    - cron: "0 16 * * *" # every day at 23:00 UTC+7
```
Comment (vansangpfiev) [inline .github/workflows/test-cortexso-model-hub.yml]: I thought we only run this CI on weekend?

File: .github/workflows/test-cortexso-model-hub.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+          sed -i 's/huggingFaceToken: ""/huggingFaceToken: "${{ secrets.HUGGINGFACE_TOKEN_READ }}"/' ~/.cortexrc
+
+      - name: Run e2e tests
+        run: |
+          cd engine
+          cp build/cortex build/cortex-nightly
+          cp build/cortex build/cortex-beta
+          python -m pip install --upgrade pip
+          python -m pip install -r e2e-test/requirements.txt
+          python e2e-test/main.py
```
Comment (vansangpfiev) [inline .github/workflows/test-cortexso-model-hub.yml]: I think can skip `main.py`

**Review by vansangpfiev — APPROVED** [review]

**Review by dan-menlo — DISMISSED** [review]
@hiento09 I think it is better for us to test the `default` model (need to check with @gabrielle-ong @namchuai on what the label is). I think in some cases, it is not the `gguf` tag. 

See the "default" below

image

**Comment by hiento09** [comment]
> @hiento09 I think it is better for us to test the `default` model (need to check with @gabrielle-ong @namchuai on what the label is). I think in some cases, it is not the `gguf` tag.
> 
> See the "default" below
> 
> image

I totally agree, @dan-homebrew

**Pull request merged**